database/rrdhost.c

```diff
@@ -707,14 +707,27 @@ void rrdhost_save_charts(RRDHOST *host) {
     rrdhost_unlock(host);
 }
 
+int is_valid_label(char *key) {
+    while(key) {
+        if(!isascii(*key) || !isdigit(*key) || !isalpha(*key) || *key != '.' || *key != '_' || *key != '-')
```

Comment:
>  Label keys must be [a-zA-Z0-9_.-]+.

no ascii there

Reply:
All these values are ascii according the function definition `checks whether c is a 7-bit unsigned char value that fits into the ASCII character set.`, but I agree that it gives the possibility to have other values, so I removed the function.

Thanks!